### PR TITLE
Adds editing a work.

### LINKED ISSUE #91: Implement editing

### PR DESCRIPTION
**Pull request by justinlittman**
closes #91

https://github.com/user-attachments/assets/1911fad5-676d-4cc5-9b9c-99f4eb6111ef

### CODE REVIEW

File: spec/support/authentication_helpers.rb [hunk 1]
```diff
@@ -15,7 +15,14 @@ def sign_in(user, groups: [])
...
   end
 
+  def clear_authentication
+    TestShibbolethHeaders.user = nil
+    TestShibbolethHeaders.groups = nil
+  end
+
   RSpec.configure do |config|
+    config.before(:each, type: :request) { clear_authentication }
+    config.before(:each, type: :system) { clear_authentication }
```
Comment (mjgiarlo) [inline spec/support/authentication_helpers.rb:25]: Can you say why we need to sign out the user before every request and system spec?

File: spec/support/authentication_helpers.rb [hunk 2]
```diff
@@ -15,7 +15,14 @@ def sign_in(user, groups: [])
     TestShibbolethHeaders.groups = groups
   end
 
+  def clear_authentication
```
Comment (mjgiarlo) [inline spec/support/authentication_helpers.rb:18]: Would you consider renaming this method `sign_out` or something similar?

File: spec/support/authentication_helpers.rb (hunk 1 above)
Comment (justinlittman) [reply spec/support/authentication_helpers.rb]: Added a comment to explain.

**Review by mjgiarlo — APPROVED** [review]
Even if we don't have a specific need for the automatic signing out (since I don't think we did this in H2?), it's a good idea to eliminate sticky state between specs. I'm 💯 in favor of preventing confusion down the line.

**Pull request merged**